yt_dlp/extractor/tubitv.py

```diff
@@ -137,7 +137,7 @@ def _real_extract(self, url):
             'subtitles': subtitles,
             'season_number': int_or_none(season_number),
             'episode_number': int_or_none(episode_number),
-            'episode': episode_title,
+            'episode': episode_title.strip() if episode_title else '',
```

Comment:
```suggestion
            'episode': strip_or_none(episode_title),
```
But isn't the problem in #10794 actually with the `title` of some episodes, which is used by default in the output template?

Reply:
Both fields were having a trailing whitespace.
I returned an explicit empty string per [this chart](https://pastebin.com/HZF5HL9g) but your suggestion is better.